roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+
+- name: Install ManageIQ service account templates
+  copy:
+    src: manageiq-serviceaccount
+    dest: /etc/origin/
+
+- name: Add Managment Infrastructure project
+  command: > 
+    {{ openshift.common.admin_binary }} new-project
+    management-infra
+    --description="Management Infrastructure"
+  register: osmiq_create_mi_project
+  failed_when: "'already exists' not in osmiq_create_mi_project.stderr and osmiq_create_mi_project.rc != 0"
+  changed_when: false
+
+- name: Create Service Account
+  command: >
+    {{ openshift.common.client_binary }} create -n
+    management-infra -f
+    /etc/origin/manageiq-serviceaccount/manageiq-serviceaccount.yaml
+  register: osmiq_create_service_account
+  failed_when: "'already exists' not in osmiq_create_service_account.stderr and osmiq_create_service_account.rc != 0"
+  changed_when: false
+
+- name: Create Cluster Role
+  command: >
+    {{ openshift.common.client_binary }} create -f
+    /etc/origin/manageiq-serviceaccount/manageiq-clusterrole.yaml
+  register: osmiq_create_cluster_role
+  failed_when: "'already exists' not in osmiq_create_cluster_role.stderr and osmiq_create_cluster_role.rc != 0"
+  changed_when: false
+
+- name: Add role to admin
+  command: >
+    {{ openshift.common.admin_binary }} policy
+    add-role-to-user -n
```

Comment:
@enoodle I'd keep the flags near the options, e.g.

    add-role-to-user
    -n management-infra
    -z managment-admin
    admin

Same for the other ones below.